kaniko.go

```diff
@@ -264,3 +283,60 @@ func getDigest(digestFile string) string {
 func trace(cmd *exec.Cmd) {
 	fmt.Fprintf(os.Stdout, "+ %s\n", strings.Join(cmd.Args, " "))
 }
+
+func getPluginEnvVars(cmdArgs []string) []string {
+	envVars := os.Environ()
+
+	// Iterate through environment variables
+	for _, envVar := range envVars {
```

Comment:
Prefer smaller if statements and continuing early. Eg-
```
for _, envVar := range envVars {
  if notValid(envVar) {
     continue
  }
  // rest of the logic here
}
```


as opposed to

```
for _, envVar := range envVars {
  if valid(envVar) {
     // bunch of logic here
  }
}
```
as it makes the code easier to read if there are smaller statements inside the if